roles/inspec/templates/etc/inspec/host-controls/attributes.yml

```diff
@@ -1,3 +1,6 @@
 {% for name,value in inspec.openstack.horizon.attributes.iteritems() %}
 {{ name }}: {{ value }}
 {% endfor %}
+{% for name,value in inspec.rhel7.attributes.iteritems() %}
```

Comment:
we should set role default to false and ensure rhosp/centos envs have inspec.enabled: true